lib/object_3d.dart

```diff
@@ -15,6 +16,9 @@ class Object3D extends StatefulWidget {
     this.color = Colors.white,
     this.object,
     this.path,
+    this.swipeCoef = 0.1,
```

Comment:
You should probably assert this is between 0 and 1 as well.

Reply:
The code clamps before use.

Reply:
This value is clamped before use between `0.001` and `0.999`

Reply:
In that same vein, do you want me to remove the exception catch block? This way all errors are thrown and the user can handle them. Also a controller would be nice to have here for these sorts of things. e.g. `objectController.hasError()` or something.

Reply:
Yeah a Widget controller is a common thing in Flutter, so if you wanted to implement this, go ahead. If not, we can also merge this PR with just the assertion. Up to you

Reply:
I want to but let's get this PR in first so we have incremental changes.